go/vt/vtgate/endtoend/vschema_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endtoend
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+	"time"
+
+	"vitess.io/vitess/go/mysql"
+)
+
+func TestVSchema(t *testing.T) {
```

Comment:
Since these are testing SrvVSchema, I suggest naming all the functions appropriately (like TestSrvVSchema)